common/spec/dependabot/pull_request_creator/message_builder/link_and_mention_sanitizer_spec.rb

```diff
@@ -22,7 +22,7 @@
       it "sanitizes the text" do
         expect(sanitize_links_and_mentions).
           to eq("<p>Great work <a href=\"https://github.com/greysteil\">"\
-            "<code>@greysteil</code></a>!</p>\n")
+            "@​greysteil</a>!</p>\n")
```

Comment:
Is the character in this string as well?

Also; wdyt about keeping the code block in there, to signal that we changed these from regular mentions?

Reply:
Ah yeah good point 👍  will pull back the code block.

Yeah wen't through and pasted in a empty space in each one to make sure 😄

Reply:
👍

Reply:
Aye yeah, maybe we can also use the `@\u200B` form for the specs? Kinda hard to spot what's going on now 😄

Reply:
> Aye yeah, maybe we can also use the `@\u200B` form for the specs? Kinda hard to spot what's going on now 😄

Yes great shout 👍